torch/ao/quantization/fx/utils.py

```diff
@@ -640,3 +651,48 @@ def get_skipped_module_name_and_classes(
         skipped_module_classes += get_custom_module_class_keys(prepare_custom_config.float_to_observed_mapping)
 
     return skipped_module_names, skipped_module_classes
+
+def _validate_qconfig_against_dtype_config(
+        qconfig: QConfig,
+        dtype_config: DTypeConfig,
+        pattern: Pattern):
+    """
+    Validate whether the QConfig satisfies all of the following constraints from the backend:
+        1. QConfig specified a quantization range that falls within the backend's, if any
+        2. QConfig specified an eps value that is <= the backend's min scale value, if any
+    """
+    def validate_activation_post_process(
+            activation_post_process_ctr: Union[_PartialWrapper, Type[ObserverBase]],
+            dtype_with_constraints: DTypeWithConstraints,
+            debug_string: str):
+        activation_post_process = activation_post_process_ctr()  # type: ignore[call-arg]
+        qconfig_qmin = getattr(activation_post_process, "quant_min", None)
+        qconfig_qmax = getattr(activation_post_process, "quant_max", None)
+        qconfig_eps = getattr(activation_post_process, "eps", None)
```

Comment:
can you explain why do we need a separate `eps`?

Reply:
I made this change based on Vasiliy's comment here: https://github.com/pytorch/pytorch/pull/85200#discussion_r974832191. Maybe we should follow offline

Reply:
As discussed offline, we will change back to `scale_min`